README.md

```diff
@@ -1,7 +1,7 @@
 [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url]
-# snappy
+# snapd
 
-Snappy is part of Ubuntu Core and enables a fully transactional Ubuntu system.
+Snapd is a tasteful, transactional packaging system designed for reliability and scalability.
```

Comment:
Let's please just repeat what we already have in the repository:

`The snapd and snap tools enable systems to work with .snap files.`